### DISCUSSION TITLE
Android and iOS Homepage Widgets

### DISCUSSION THREAD

**Discussion by BillyMartin1964** [shinyorg/shiny#1324] [Q&A]
Since homepage widgets use foreground services, can I use Shiny to create one in Maui, and how would I do it?

**Comment by aritchie** [comment]
Shiny has nothing to do with allowing a user to create a foreground service and obviously for iOS there isnt a concept of this.  Shiny modules create an Android foreground service for themselves as needed (ie GPS).
